civihr_employee_portal/src/Forms/OnboardingWizardCustomizationForm.php

```diff
@@ -45,6 +47,12 @@ public function onSubmit($form, &$formState) {
 
     $this->saveLogo($formState);
     $this->updateCarouselContent($formState);
+
+    // sends user to the first step of the onboarding wizard
+    // which is just node edit page with custom theming
+    if ($this->clickedSaveAndTest($formState)) {
+      drupal_goto('/user/' . user_uid_optional_load()->uid . '/edit', ['query' => [ 'test_onboarding_wizard' => 1 ]]);
```

Comment:
This line is very long, can you separate it into different variables, like `$userEditUrl` and `$queryParams`?

Reply:
Sure, no problem. Thanks.